jepsen/src/jepsen/checker.clj

```diff
@@ -46,8 +46,11 @@
           true
           valids))
 
+;; DEPRECATED Checkers should be implemented without model. Model should be an
```

Comment:
I'd put this comment in the docstring for `(check ...)`

Reply:
done